modules/installation-vsphere-infrastructure.adoc

```diff
@@ -45,47 +45,44 @@ endif::[]
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 6 or 7 instance that meets the requirements for the components that you use.
+You must install the {product-title} cluster on a VMware vSphere version 7 instance that meets the requirements for the components that you use.
 
 
 .Version requirements for vSphere virtual environments
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
-|VM hardware version | 13 or later
-|vSphere ESXi hosts | 6.5 or later
-|vCenter host   | 6.5 or later
+|VM hardware version | 15 or later
+|vSphere ESXi hosts | 7.0.2 or later
+|vCenter host   | 7.0.2 or later
```

Comment:
It's very confusing to have three PRs that have very similar information (#47785, #46711).

`vSphere ESXi hosts` and `vCenter host`  are **required** to be at least 6.7u3 and 7.0.2 is **strongly** recommended. Anything lower than 7.0.2 is deprecated in 4.11 (but still supported, the text below is OK).

Reply:
The information in the release notes is relevant from both teams' perspectives. The change to the vSphere infrastructure requirements table here applies to different branches than the release notes and needs a different PR.

Reply:
@kalexand-rh  @jsafrane 
I think the **7.0.2** in 1st commit is ok as we mentioned 7.0.1 or earlier is now deprecated in the text below. 
In the 2nd commit now it changes to 6.7.u3, I think we should switch back to 7.0.2 to keep align with [#47785](https://github.com/openshift/openshift-docs/pull/47785 )